### PR TITLE
advise to user on iof_file_only usage

### PR DESCRIPTION
**Pull request by naughtont3**
Signed-off-by: Thomas Naughton <naughtont@ornl.gov>

### CODE REVIEW

File: Chap_API_Tools.tex [hunk 1]
```diff
@@ -313,6 +313,14 @@ \subsection{Direct launch}
     \item \pasteAttributeItem{PMIX_DEBUG_WAIT_FOR_NOTIFY}
 \end{itemize}
 
+
+\adviceuserstart
+The \refattr{PMIX_IOF_FILE_ONLY} indicates output is directed to files and
+no copy is sent back to the application.  For example, this can be combined with
+\refattr{PMIX_IOF_OUTPUT_TO_FILE} or \refattr{PMIX_IOF_OUTPUT_TO_FILE} to
```
Comment (rhc54) [inline Chap_API_Tools.tex]: One of these should be "output_to_directory", yes?

Comment (naughtont3) [reply Chap_API_Tools.tex]: LOL.  Yes

Comment (rhc54) [reply Chap_API_Tools.tex]: Just updated the branch based on your comments - minus this change. You may need to rebase (I don't think so, but possible).

Comment (naughtont3) [reply Chap_API_Tools.tex]: OK, fixed typo.  I think it should merge to your branch cleanly.  If problems, please just copy/paste.  Thanks Ralph.

**Review by rhc54 — APPROVED** [review]
Good - thx!

**Pull request merged**